inc/compatibility/class-compatibilityfactory.php

```diff
@@ -0,0 +1,567 @@
+					),
+					esc_url( 'https://docs.pantheon.io/plugins-known-issues' )
+				);
+				?>
+			</p>
+
+			<div id="health-check-compatibility" class="health-check-accordion">
+
+				<?php
+
+				$info = [
+					'automatic' => [
+						'label' => __( 'Automatic Fixes' ),
+						'description' => __( 'Compatibility with the following plugins has been automatically added.' ),
+						'fields' => get_option( 'pantheon_applied_fixes' ),
+						'show_count' => true,
+					],
+					'manual' => [
+						'label' => __( 'Manual Fixes' ),
+						'description' => __( 'Compatibility with the following plugins needs to be manually applied.' ),
+						'fields' => $this->get_manual_fixes(),
+						'show_count' => true,
+					],
+					'notes' => [
+						'label' => __( 'Needs Review' ),
+						'description' => __( 'Compatibility with the following plugins needs to be reviewed.' ),
+						'fields' => $this->get_review_fixes(),
+						'show_count' => true,
+					],
+				];
+				foreach ( $info as $section => $details ) :
+					if ( empty( $details['fields'] ) ) {
+						continue;
+					}
+
+					?>
+					<h3 class="health-check-accordion-heading">
+						<button aria-expanded="false" class="health-check-accordion-trigger" aria-controls="health-check-accordion-block-<?php echo esc_attr( $section ); ?>" type="button">
+					<span class="title">
+						<?php echo esc_html( $details['label'] ); ?>
+						<?php
+
+						if ( isset( $details['show_count'] ) && $details['show_count'] ) {
+							printf(
+								'(%s)',
+								esc_html( number_format_i18n( count( $details['fields'] ) ) )
+							);
+						}
+
+						?>
+					</span>
+							<span class="icon"></span>
+						</button>
+					</h3>
+
+					<div id="health-check-accordion-block-<?php echo esc_attr( $section ); ?>" class="health-check-accordion-panel" hidden="hidden">
+						<?php
+
+						if ( ! empty( $details['description'] ) ) {
+							printf( '<p>%s</p>', esc_html( $details['description'] ) );
+						}
+
+						?>
+						<table class="widefat striped health-check-table" role="presentation">
+							<thead>
+							<tr>
+								<th><?php esc_html_e( 'Plugin' ); ?></th>
+								<th><?php esc_html_e( 'Compatibility Status' ); ?></th>
+							</tr>
+							</thead>
+							<tbody>
+							<?php
+
+							foreach ( $details['fields'] as $field ) {
+								$values = '<ul>';
+								$values .= '<li><b>' . ucfirst( $field['plugin_status'] ) . '</b></li>';
+								$values .= '<li>' . $field['plugin_message'] . '</li>';
+								$values .= '</ul>';
+								printf(
+									'<tr><td>%s</td><td>%s</td></tr>',
+									esc_html( $field['plugin_name'] ),
+									wp_kses_post( $values )
+								);
+							}
+
+							?>
+							</tbody>
+						</table>
+					</div>
+				<?php endforeach; ?>
+			</div>
+		</div>
+		<?php
+	}
+
+	private function get_manual_fixes() {
```

Comment:
Is there a way to detect if the manual fixes are applied and hid (or succeed) the Site Health test if so? Is that already happening somewhere else?